### PR TITLE
Adjust contract/blockchain syscalls

### LINKED ISSUE #1031: Adjust Blockchain and Contract syscalls to Neo 3
```
GetHeight = Register("System.Blockchain.GetHeight", Blockchain_GetHeight, 0_00000400, TriggerType.Application, CallFlags.None);
GetBlock = Register("System.Blockchain.GetBlock", Blockchain_GetBlock, 0_02500000, TriggerType.Application, CallFlags.None);
GetTransaction = Register("System.Blockchain.GetTransaction", Blockchain_GetTransaction, 0_01000000, TriggerType.Application, CallFlags.None);
GetTransactionHeight = Register("System.Blockchain.GetTransactionHeight", Blockchain_GetTransactionHeight, 0_01000000, TriggerType.Application, CallFlags.None);
GetTransactionFromBlock = Register("System.Blockchain.GetTransactionFromBlock", Blockchain_GetTransactionFromBlock, 0_01000000, TriggerType.Application, CallFlags.None);
GetContract = Register("System.Blockchain.GetContract", Blockchain_GetContract, 0_01000000, TriggerType.Application, CallFlags.None);
Create = Register("System.Contract.Create", Contract_Create, GetDeploymentPrice, TriggerType.Application, CallFlags.AllowModifyStates);
Update = Register("System.Contract.Update", Contract_Update, GetDeploymentPrice, TriggerType.Application, CallFlags.AllowModifyStates);
Destroy = Register("System.Contract.Destroy", Contract_Destroy, 0_01000000, TriggerType.Application, CallFlags.AllowModifyStates);
Call = Register("System.Contract.Call", Contract_Call, 0_01000000, TriggerType.System | TriggerType.Application, CallFlags.AllowCall);
CallEx = Register("System.Contract.CallEx", Contract_CallEx, 0_01000000, TriggerType.System | TriggerType.Application, CallFlags.AllowCall);
IsStandard = Register("System.Contract.IsStandard", Contract_IsStandard, 0_00030000, TriggerType.All, CallFlags.None);
CreateStandardAccount = Register("System.Contract.CreateStandardAccount", Contract_CreateStandardAccount, 0_00010000, TriggerType.All, CallFlags.None);
```
`CreateStandardAccount` and `CallEx` were added, `GetTransaction` and `GetBlock` are separate tasks (see #1023 and #1025), everything else needs to be adjust in terms of prices/flags/types/names.

### PR DESCRIPTION
**Pull request by fyrchik**
Closes #1031 .

### CODE REVIEW

File: pkg/core/interop_system.go [hunk 1]
```diff
@@ -470,3 +470,19 @@ func contractDestroy(ic *interop.Context, v *vm.VM) error {
...
+
+// contractIsStandard checks if contract is standard (sig or multisig) contract.
+func contractIsStandard(ic *interop.Context, v *vm.VM) error {
+	h := v.Estack().Pop().Bytes()
+	u, err := util.Uint160DecodeBytesBE(h)
+	if err != nil {
+		return err
+	}
+	var result bool
+	cs, err := ic.DAO.GetContractState(u)
```
Comment (roman-khimov) [inline pkg/core/interop_system.go:483]: `err` is not used.

File: pkg/core/interop_system.go [hunk 2]
```diff
@@ -470,3 +470,19 @@ func contractDestroy(ic *interop.Context, v *vm.VM) error {
 	}
 	return nil
 }
+
+// contractIsStandard checks if contract is standard (sig or multisig) contract.
+func contractIsStandard(ic *interop.Context, v *vm.VM) error {
```
Comment (roman-khimov) [inline pkg/core/interop_system.go:476]: This one is not tested.

File: pkg/core/interop_system.go (hunk 1 above)
Comment (fyrchik) [reply pkg/core/interop_system.go:483]: Fixed.

File: pkg/core/interop_system.go (hunk 2 above)
Comment (fyrchik) [reply pkg/core/interop_system.go:476]: Fixed.

**Pull request merged**